src/main/resources/application.yml

```diff
@@ -2,6 +2,11 @@ info.app:
   name: Hmpps Incentives API
   version: 1.0
 
+task:
+  update-kpis:
+    # Every 1st of the month at 3:00am
```

Comment:
oh actually sounds v sensible to do the "how many overdue reviews?" check at night cos nobody is part-way through whatever they've planned for the day.

Reply:
Yeah, and it's a snapshot anyway so as long as it's a number taken around the same time it doesn't really matter the precise time when it's taken.

BTW: I had a chat with Steve this morning and he concurs. He's happy with this and he agrees it doesn't have to be taken separately on the last day of the month.